index.html

```diff
@@ -21,5 +21,8 @@ <h1 id="goButton" class="button">Go</h1>
       <div id="goLight" class="bulb"></div>
     </div>
 
+
+    <script src="js/jquery-3.4.0.js"></script>
+    <script src="js/app.js"></script>
```

Comment:
@jbwaldrop01 I couldn't test out your js because I noticed that the js file and the title on here don't match.